posts/maxistahl/03.design-system-post-1.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Building a Unified Design System Language
+published: true
+description: Uniting UI-UX and Dev Teams through shared terminology and optimal workflows in Design System development
+tags: 'design, react, webdev, javascript'
+cover_image: ./assets/design-system-post-1/cover.png
+id: 1499474
+series: 'Design Systems: The Next Chapter'
+---
+
+The frontend job can be defined as creating a visual interface using code. However, this definition is basic and doesn't capture the full complexity of modern frontend development.
```

Comment:
```suggestion
Frontend job can be defined as creating a visual interface using code. However, this definition is basic and doesn't capture the full complexity of modern frontend development.
```

I would use `whole` instead of `full`, but neither of them are wrong